package.json

```diff
@@ -52,9 +52,9 @@
     "vscode-test-utils": "^0.0.8"
   },
   "scripts": {
-    "precompile": "node ./node_modules/vscode/bin/install",
-    "compile": "tsc",
-    "lint": "tslint --project tsconfig.json",
+    "postinstall": "node ./node_modules/vscode/bin/install",
```

Comment:
I had issues w/ other projects if they were a dependency of another project, this would fail upon install, but considering this is an extension and shouldn't be a dependency of another project, I'll accept this.